server/server.go

```diff
@@ -582,10 +615,15 @@ func vec64To32(vec3 mgl64.Vec3) mgl32.Vec3 {
 // itemEntries loads a list of all custom item entries of the server, ready to be sent in the StartGame
 // packet.
 func (server *Server) itemEntries() (entries []protocol.ItemEntry) {
-	for name, rid := range itemRuntimeIDs {
+	for _, it := range world.Items() {
```

Comment:
This re-introduces the issue where not all vanilla items are sent to the client causing inventory glitches. Do you reckon we could somehow fix this? Not sure exactly how the custom items get assigned runtime IDs, but if they're just appended we could just start counting from the end of `itemRuntimeIDs`?

Reply:
The runtime IDs do seem to start right at the end of `itemRuntimeIDs`, so this should be possible.